tiered-storage/jcloud/src/test/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreManagedLedgerOffloaderBase.java

```diff
@@ -82,7 +71,11 @@ protected static MockZooKeeper createMockZooKeeper() throws Exception {
                 CreateMode.PERSISTENT);
         return zk;
     }
-    
+
+    protected static MockManagedLedger createMockManagedLedger() {
```

Comment:
Can you explain why do you add a "Mock" method in the actual implementation?

Reply:
`BlobStoreManagedLedgerOffloaderBase` is a class for test, not implementation. Its name doesn't like a test class, but you can see its directory `src/test/java/org/apache/bookkeeper/mledger/offload/jcloud/impl/BlobStoreManagedLedgerOffloaderBase.java` is a test directory.